### PR TITLE
Ci 496 update int home page

### PR DESCRIPTION
**Pull request by jessgilbert**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if adding CSS) CSS have been compiled.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [x] (if changing the UI) Add a printscreen to the PR

screencapture-international-trade-great-8012-international-2019-09-16-10_50_21

### CODE REVIEW

File: core/sass/pages/international/_landing-page.scss [hunk 1]
```diff
@@ -7,54 +7,6 @@
 
 .landing-page {
 
-  @mixin landingPageSection {
```
Comment (MichaelWalker) [inline core/sass/pages/international/_landing-page.scss]: excellent! More deleting things!

File: core/tests/test_templates.py [hunk 2]
```diff
@@ -17,43 +17,6 @@
 }
 
 
-def test_homepage_button_how_to_do_business_feature_off(default_context):
```
Comment (MichaelWalker) [inline core/tests/test_templates.py]: We don't seem to have changed anything related to these tests? Am I missing something?

Comment (jessgilbert) [reply core/tests/test_templates.py]: In the landing page hero it needed this feature on for some reason

File: core/templates/core/includes/hero_landing_page.html [hunk 3]
```diff
@@ -1,13 +0,0 @@
-{% load static %}
-
-<section class="hero with-flag" data-ga-section="hero" style="background-image: url('{{ img_url }}')">
-  <div class="container">
-    <div class="hero-text">
-      <h1 class="heading-xlarge margin-bottom-15">{{ heading }} <br> {{ subtitle }}</h1>
-
-      {% if cta_text and features.HOW_TO_DO_BUSINESS_ON %}
```
Comment (jessgilbert) [inline core/templates/core/includes/hero_landing_page.html]: @MichaelWalker here

File: core/tests/test_templates.py (hunk 2 above)
Comment (MichaelWalker) [reply core/tests/test_templates.py]: haha - yep. Totally missed that deleted file! Thanks

**Pull request merged**